diff -r a55394b9754b Makefile.in
--- a/Makefile.in	Wed Aug 22 11:41:31 2007 +0300
+++ b/Makefile.in	Wed Aug 22 11:57:01 2007 +0300
@@ -42,8 +42,8 @@ exec_prefix = @exec_prefix@
 exec_prefix = @exec_prefix@
 transform = @program_transform_name@
 bindir = $(exec_prefix)/bin
-infodir = $(prefix)/info
-mandir = $(prefix)/man/man1
+infodir = $(prefix)/share/info
+mandir = $(prefix)/share/man/man1
 manext = .1
 
 COMPILE = $(CC) -c $(DEFS) $(INCLUDES) $(CPPFLAGS) $(CFLAGS)
@@ -129,13 +129,13 @@ check: wdiff
 check: wdiff
 	$(srcdir)/check_it
 
-install: all
-	$(srcdir)/mkinstalldirs $(bindir) $(infodir)
+install: all install-man
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(infodir)
 	for name in $(PROGRAMS); do \
 	  $(INSTALL_PROGRAM) $$name \
-		$(bindir)/`echo $$name | sed '$(transform)'`; \
-	done
-	$(INSTALL_DATA) $(srcdir)/wdiff.info $(infodir)/wdiff.info
+		$(DESTDIR)$(bindir)/`echo $$name | sed '$(transform)'`; \
+	done
+	$(INSTALL_DATA) $(srcdir)/wdiff.info $(DESTDIR)$(infodir)/wdiff.info
 
 uninstall:
 	for name in $(PROGRAMS); do
@@ -147,9 +147,9 @@ uninstall:
 # GNU and switch to Info.  The following is for irreductible fellows.
 
 install-man:
-	$(srcdir)/mkinstalldirs $(mandir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)
 	$(INSTALL_DATA) $(srcdir)/wdiff.1 \
-		$(mandir)/`echo wdiff | sed '$(transform)'`$(manext)
+		$(DESTDIR)$(mandir)/`echo wdiff | sed '$(transform)'`$(manext)
 
 uninstall-man:
 	rm -f $(mandir)/`echo wdiff | sed '$(transform)'`$(manext); \
